src/manager/FileBackedTaskManager.java

```diff
@@ -0,0 +1,220 @@
+package manager;
+
+import task.Epic;
+import task.SubTask;
+import task.Task;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.nio.file.Files;
+import java.util.ArrayList;
+
+public class FileBackedTaskManager extends InMemoryTaskManager {
+
+    File file;
+
+
+    public FileBackedTaskManager(File file) {
+        this.file = file;
+    }
+
+
+    private void save() {
+
+        try {
+            if (!Files.exists(file.toPath())) {
+                Files.createFile(file.toPath());
+            }
+
+            ArrayList<String> taskCSVLinesArray = new ArrayList<>();
```

Comment:
Переменную `taskCSVLinesArray` следует объявить через интерфейс `List`, а не конкретную реализацию `ArrayList`, чтобы код был более гибким. Это упростит замену реализации в будущем и повысит читаемость
```suggestion
            List<String> taskCSVLinesArray = new ArrayList<>();
```